src/views/menus/AccountMenu.tsx

```diff
@@ -166,18 +168,23 @@ export const AccountMenu = () => {
                 />
               </WithTooltip>
             </$AddressRow>
-            {walletType && ![WalletType.Privy, WalletType.Keplr].includes(walletType) && (
-              <$AddressRow>
-                <div tw="relative z-[1] rounded-[50%] bg-[#303045] p-0.375 text-[1rem] leading-[0]">
-                  <Icon iconName={IconName.AddressConnector} tw="absolute top-[-1.625rem] h-1.75" />
-                  <Icon iconComponent={wallets[walletType].icon as ElementType} />
-                </div>
-                <$Column>
-                  <$label>{stringGetter({ key: STRING_KEYS.SOURCE_ADDRESS })}</$label>
-                  <$Address>{address}</$Address>
-                </$Column>
-              </$AddressRow>
-            )}
+            {connectedWallet &&
+              connectedWallet.name !== WalletType.Privy &&
+              connectedWallet.name !== WalletType.Keplr && (
+                <$AddressRow>
+                  <div tw="relative z-[1] rounded-[50%] bg-[#303045] p-0.375 text-[1rem] leading-[0]">
```

Comment:
I know you didn't add this but just noting in a follow up that we should add the `bg-[#303045]` to our color theme